backend/api/Controllers/RobotController.cs

```diff
@@ -643,34 +643,40 @@ public async Task<ActionResult> ResumeMission([FromRoute] string robotId)
     /// </remarks>
     [HttpPost]
     [Authorize(Roles = Role.User)]
-    [Route("{robotId}/start-localization")]
+    [Route("start-localization")]
     [ProducesResponseType(typeof(Mission), StatusCodes.Status200OK)]
     [ProducesResponseType(StatusCodes.Status400BadRequest)]
     [ProducesResponseType(StatusCodes.Status401Unauthorized)]
     [ProducesResponseType(StatusCodes.Status403Forbidden)]
     [ProducesResponseType(StatusCodes.Status404NotFound)]
     [ProducesResponseType(StatusCodes.Status500InternalServerError)]
     public async Task<ActionResult<Mission>> StartLocalizationMission(
-        [FromRoute] string robotId,
-        [FromBody] Pose localizationPose
```

Comment:
Don't remove localizationPose as input. It's used in https://github.com/equinor/flotilla/pull/870

Reply:
I added this in the new class ScheduleLocalizationMissionQuery.cs